### PR TITLE
Docs updated

### PR DESCRIPTION
**Pull request by Brambleberry4**
Created more recent docs and tried to update the existing ones.

### CODE REVIEW

File: source/preferences.rst [hunk 1]
```diff
@@ -440,7 +440,7 @@ Command
 
 
 ^^^^^^^
-Colours
+Colors
```
Comment (mattrose) [inline source/preferences.rst:443]: Colours vs. Colors is a British vs American Spelling thing, and we've kind of settled on British spelling, as I'm Canadian and follow British spelling conventions, and all the previous authors were from the UK.

**Review by mattrose — CHANGES_REQUESTED** [review]
Other than the color vs colour thing, this all looks great.

File: source/preferences.rst (hunk 1 above)
Comment (Brambleberry4) [reply source/preferences.rst]: I fixed it

**Comment by mattrose** [comment]
Thanks, now I just have to figure out how to update the readthedocs site.

**Pull request merged**